src/assets/translations/messages-en.json

```diff
@@ -670,6 +670,9 @@
   "accounts": {
     "accounts": "Accounts"
   },
+  "feedbacks": {
```

Comment:
remove this `feedbacks` key, put `marketDataUnavailable` under the `assets` key above